File: Database/Entities/Book.cs
namespace Database.Entities;

public class Book
{
    public int Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    
    public byte[]? Image { get; set; }
    public double Rating { get; set; }
    public DateTime DateAdded { get; set; }
    public bool available { get; set; }
    
    public ICollection<BookAuthor> BookAuthors { get; set; }
}

File: client/src/app/Models/book.ts
import {BookAuthor} from "./book-author";

export interface Book {

  id: number;
  title: string;
  description: string;
  image: any;
  rating: number;
  dateAdded: Date;
  available: boolean;
  authors: any[];
  imageUrl?: string;
  imageBase64?: string;
}
